src/main/java/org/isf/login/rest/LoginController.java

```diff
@@ -49,10 +52,11 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import io.jsonwebtoken.JwtException;
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
-@RestController(value = "/auth")
```

Comment:
@SilverD3 the value parameter for @RestController annotation is not really needed, should we remove it in all controllers?

Reply:
Yes, it has no effect. I suppose when it was added, it was supposed to behave like `@RequestMapping(value = "/auth")`.

Reply:
yes, and it could even bring problem as far as I'm understanding, don't you mind to open an issue to remove all of them?

Reply:
Yes, you can definitely remove them, I mean if it's okay for @dbmalkovsky too.

Reply:
Not in this PR anyway

Reply:
Created OH2-403 for this.